### DISCUSSION TITLE
Watchdog time increase

### DISCUSSION THREAD

**Discussion by lukasa1993** [HomeSpan/HomeSpan#966] [Q&A]
how can i increase watchdog time to like a minute when i am using autopoll ?

**Comment by HomeSpan** [comment]
HomeSpan does not use the watchdog timer.  Exactly what error are you encountering?

**Comment by lukasa1993** [comment]
nvm split the action so it doesn't hit watchdog if homespan doesn't have it some other lib has it probably
